### DISCUSSION TITLE
Deleting a parent node should delete all child nodes

### DISCUSSION THREAD

**Discussion by ghost** [SkriptLang/Skript#4031] [Suggestions]
EDIT: Please disregard this as after some experimentation I've realized that the behavior described here already exists and the opposite happening is potentially a bug that I'm trying to find a way to reproduce 100% of the time. In any case an issue would be open and it does not belong here in the discussions page.

----

So, right now let's say you have the following nested values:
```
{foo::bar::zee::wuu}
```

And you do:
```
delete {foo::*}
```
The zee and wuu values are still accesible. Instead Skript should automatically clear all child nodes that have lost the reference link.

**Comment by Romitou** [comment]
I just tested and it seems to me that this is already the right behavior, isn't it?
![WindowsTerminal_KxCQCL0MKq](https://bild.romitou.fr/contents/3D-F2mV1hI.png)
Equivalent in skript code:
```applescript
command test:
    trigger:
        set {foo::bar::zee::wuu} to true
        broadcast {foo::bar::zee::wuu} # returns true
        delete {foo::*}
        broadcast {foo::bar::zee::wuu} # returns <none>
```
